public/index.html

```diff
@@ -0,0 +1,322 @@
+<!doctype html>
```

Comment:
Is it worth having these generated static files here? It could be handy, but then it also risks having old / stale files in here. Might be better to be forced to generate these as part of the build pipeline.

Reply:
I can try making it part of the docker image build process

Reply:
I have deleted the generated files from `/public` these are now added as part of the docker build process